ethportal-peertest/src/scenarios/offer_accept.rs

```diff
@@ -68,3 +72,55 @@ pub fn test_offer_accept(peertest_config: PeertestConfig, peertest: &Peertest) {
         "The received content {received_content_str}, must match the expected {HISTORY_CONTENT_VALUE}",
     );
 }
+
+pub async fn test_populate_offer(peertest_config: PeertestConfig, peertest: &Peertest) {
+    info!("Testing Populate Offer/ACCEPT flow");
```

Comment:
`test_populated_offer` & `Testing Populated Offer/Accept flow`? And rename previous test to `test_unpopulated_offer`?